.env.development

```diff
@@ -5,5 +5,5 @@ VUE_APP_API_PUBLIC_URL=https://public.int.bgdi.ch/
 VUE_APP_DATA_BASE_URL=https://data.geo.admin.ch/
 VUE_APP_WMTS_BASE_URL=https://wmts{5-9}.int.bgdi.ch/
 VUE_APP_WMS_BASE_URL=https://wms-bgdi-cdn-0.int.bgdi.ch/
-VUE_APP_API_SERVICE_ICON_BASE_URL=https://service-icons.bgdi-dev.swisstopo.cloud/v4/
+VUE_APP_API_SERVICE_ICON_BASE_URL=https://sys-map.dev.bgdi.ch/api/
```

Comment:
@pakb 
If I'm not mistaken, there are multiple services which will require this base url (feedback, qrcode, icons). Should we directly change the variable name so that when we want to include these services, we don't reach them via the service icon base url ^^ ?